src/login/app_utils/validation/emptyValueValidation.js

```diff
@@ -0,0 +1,10 @@
+import { emptyStringPattern } from "./regexPatterns";
+
+export const emptyValueValidation = (values) => {
+  const errors = {};
+  const groupName = values.groupName;
+  if (!groupName || emptyStringPattern.test(groupName)) {
```

Comment:
I saw that you used trim function for sending data to backend. However, I would recommend to use `!groupName.trim()` instead of regexPattern.

Reply:
After our discussion this morning about the use of this in PR521, I will leave this as is.

Reply:
:+1: